tests/js/spec/views/dashboardsV2/sortableWidget.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
+import {initializeOrg} from 'sentry-test/initializeOrg';
+
+import SortableWidget from 'app/views/dashboardsV2/sortableWidget';
+import {DisplayType, WidgetType} from 'app/views/dashboardsV2/types';
+
+describe('Dashboards > SortableWidget', () => {
+  const organization = TestStubs.Organization({
+    features: ['dashboards-basic', 'dashboards-edit'],
+  });
+  const widget = {
+    title: 'Test Query',
+    displayType: DisplayType.LINE,
+    widgetType: WidgetType.DISCOVER,
+    interval: '5m',
+    queries: [
+      {
+        name: '',
+        conditions: '',
+        fields: ['count()'],
+        orderby: '',
+      },
+    ],
+  };
+
+  let initialData;
+
+  beforeEach(() => {
+    initialData = initializeOrg({organization, router: {}, project: 1, projects: []});
+  });
+  it('renders a discover WidgetCard', async () => {
+    const wrapper = mountWithTheme(
+      <SortableWidget
+        widget={widget}
+        dragId=""
+        isEditing={false}
+        onDelete={() => undefined}
+        onEdit={() => undefined}
+      />,
+      initialData.routerContext
+    );
+    expect(wrapper.find('IssueWidgetCard').length).toBe(0);
+    expect(wrapper.find('WidgetCard').length).toBe(1);
+  });
+  it('renders an IssueWidgetCard', async () => {
```

Comment:
This test asserts `expect(wrapper.find('IssueWidgetCard').length).toBe(0);`, I think you intended to update the `widgetType` to be for an Issues widget and assert the length is 1?

Reply:
😅  good catch, forgot to update this! Done!